src/SAML2/SignedElementHelper.php

```diff
@@ -15,54 +16,41 @@
  *
  * @package SimpleSAMLphp
  */
-class SignedElementHelper extends SignedElement
+trait SignedElementHelper
 {
     /**
-     * Available methods for validating this message.
-     *
-     * @var array
-     */
-    private $validators = [];
-
-    /**
-     * How long this element is valid, as a unix timestamp.
+     * SignedElement constructor.
      *
-     * @var int|null
+     * @param XMLSecurityDSig|null $signature
```

Comment:
@var was correct here..   And again, I think your IDE is 'optimizing' the namespaces away, but it's not what we do throughout the rest of the project